js/bithumb.js

```diff
@@ -391,46 +396,53 @@ module.exports = class bithumb extends Exchange {
-        //             },
-        //             "date":"1587710878669"
-        //         }
-        //     }
-        //
         const result = {};
-        const data = this.safeValue (response, 'data', {});
-        const timestamp = this.safeInteger (data, 'date');
-        const tickers = this.omit (data, 'date');
-        const ids = Object.keys (tickers);
-        for (let i = 0; i < ids.length; i++) {
-            const id = ids[i];
-            let symbol = id;
-            let market = undefined;
-            if (id in this.markets_by_id) {
-                market = this.markets_by_id[id];
-                symbol = market['symbol'];
+        const fetchTickersOptions = this.safeValue (this.options, 'fetchTickers', {});
+        let quoteCurrencies = this.safeValue (fetchTickersOptions, 'quoteCurrencies', {});
+        quoteCurrencies = this.safeValue (params, 'quoteCurrencies', quoteCurrencies);
+        const quoteKeys = Object.keys (quoteCurrencies);
+        for (let i = 0; i < quoteKeys.length; i++) {
+            const quoteCurrency = quoteKeys[i];
+            if (!quoteCurrencies[quoteCurrency]) {
+                continue;
             }
-            const ticker = tickers[id];
-            const isArray = Array.isArray (ticker);
-            if (!isArray) {
-                ticker['date'] = timestamp;
-                result[symbol] = this.parseTicker (ticker, market);
+            const method = 'publicGetTickerALL' + quoteCurrency;
+            const response = await this[method] (params);
+            //
+            //     {
+            //         "status":"0000",
+            //         "data":{
+            //             "BTC":{
+            //                 "opening_price":"9045000",
+            //                 "closing_price":"9132000",
+            //                 "min_price":"8938000",
+            //                 "max_price":"9168000",
+            //                 "units_traded":"4619.79967497",
+            //                 "acc_trade_value":"42021363832.5187",
+            //                 "prev_closing_price":"9041000",
+            //                 "units_traded_24H":"8793.5045804",
+            //                 "acc_trade_value_24H":"78933458515.4962",
+            //                 "fluctate_24H":"530000",
+            //                 "fluctate_rate_24H":"6.16"
+            //             },
+            //             "date":"1587710878669"
+            //         }
+            //     }
+            //
+            const data = this.safeValue (response, 'data', {});
+            const timestamp = this.safeInteger (data, 'date');
+            const tickers = this.omit (data, 'date');
+            const ids = Object.keys (tickers);
+            for (let i = 0; i < ids.length; i++) {
+                const id = ids[i];
+                const ticker = tickers[id];
+                const symbol = id + '/' + quoteCurrency;
+                const market = this.market (symbol);
```

Comment:
We must use `safeSymbol` or `safeMarket` in the parsing code:

```suggestion
                const marketId = id + '/' + quoteCurrency; // <<<< if you think of this line a bit more, you will come to realize that bithumb's fetchMarkets is broken at this time and needs fixing even before fixing these tickers
                const market = this.safeMarket (symbol);
```